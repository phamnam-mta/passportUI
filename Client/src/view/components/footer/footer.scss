@import "~@fluentui/react/dist/sass/References";

.footer-container {
    display: flex;
    width: 100%;
    padding: 10px 0;
    border-top: 1px solid $ms-color-gray30;
    justify-content: flex-end;
    background-color: $ms-color-white;
    align-items: center;

    .privacy-link {
        color:$ms-color-black;
        text-decoration: none;

        &:hover {
            text-decoration: underline;
        }
    }
    .copyright-year {
        margin: 0 20px;
        line-height: 0;
    }
}