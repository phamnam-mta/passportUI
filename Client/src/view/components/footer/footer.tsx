import * as React from "react";

import "./footer.scss";

export const Footer = () => {
    return (
        <footer className="footer-container">
            <p className="copyright-year">&copy; 2022</p>
        </footer>
    );
};

export default Footer;
