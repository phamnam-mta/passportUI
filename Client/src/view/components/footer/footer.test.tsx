import { shallow } from "enzyme";
import * as React from "react";
import { Footer } from "./footer";

describe("<Footer />", () => {
    describe("Rendering", () => {
        it("should render correctly", () => {
            const wrapper = shallow(<Footer />);
            expect(wrapper).toMatchSnapshot();
        });
    });
});
