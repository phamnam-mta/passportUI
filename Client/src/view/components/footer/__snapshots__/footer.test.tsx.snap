// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Rendering should render correctly 1`] = `
<footer
  className="footer-container"
>
  <p
    className="copyright-year"
  >
    Â© 2022
  </p>
</footer>
`;
